internal/connector/connector.go

```diff
@@ -297,23 +297,26 @@ func httpTransportFromOptions(opts ServerOptions) *http.Transport {
 func syncDestination(con connector) error {
 	endpoint, err := getEndpointHostPort(con.k8s, con.options)
 	if err != nil {
-		return err
+		logging.L.Warn().Err(err).Msg("could not get host")
```

Comment:
maybe some more info here on why not getting the host could be happening would be useful, looks like something that is unexpected otherwise